frontend/src/scenes/session-recordings/player/inspector/components/PlayerInspectorListItem.tsx

```diff
@@ -0,0 +1,197 @@
+                                            or otherwise not able to be linked to a given window.
+                                        </>
+                                    )}
+                                </>
+                            ) : null}
+                        </>
+                    }
+                >
+                    <div className="shrink-0 text-2xl h-8 text-muted-alt flex items-center justify-center gap-1">
+                        {showIcon && TypeIcon ? <TypeIcon /> : null}
+                        {windowNumber ? <IconWindow size="small" value={windowNumber} /> : null}
+                    </div>
+                </Tooltip>
+            )}
+
+            <span
+                className={clsx(
+                    'flex-1 overflow-hidden rounded border',
+                    isExpanded && 'border-primary',
+                    item.highlightColor && `border-${item.highlightColor}-dark bg-${item.highlightColor}-highlight`,
+                    !item.highlightColor && 'bg-inverse'
+                )}
+            >
+                {item.type === SessionRecordingPlayerTab.NETWORK ? (
+                    <ItemPerformanceEvent item={item.data} finalTimestamp={end} {...itemProps} />
+                ) : item.type === SessionRecordingPlayerTab.CONSOLE ? (
+                    <ItemConsoleLog item={item} {...itemProps} />
+                ) : item.type === SessionRecordingPlayerTab.EVENTS ? (
+                    <ItemEvent item={item} {...itemProps} />
+                ) : null}
+
+                {isExpanded ? (
+                    <div className="text-xs">
+                        <LemonDivider dashed />
+
+                        <div
+                            className="flex gap-2 justify-end cursor-pointer m-2"
+                            onClick={() => setItemExpanded(index, false)}
+                        >
+                            <span className="text-muted-alt">Collapse</span>
+                        </div>
+                    </div>
+                ) : null}
+            </span>
+            {!isExpanded ? (
+                <LemonButton size="small" noPadding status="primary-alt" onClick={() => seekToEvent()}>
+                    <span className="p-1 text-xs">
+                        {timestampMode === 'absolute' ? (
+                            <TZLabel time={item.timestamp} formatDate="DD, MMM" formatTime="HH:mm:ss" noStyles />
+                        ) : (
+                            <>
+                                {item.timeInRecording < 0 ? (
+                                    <Tooltip
+                                        title="This event occured before the recording started, likely as the page was loading."
+                                        placement="left"
+                                    >
+                                        {colonDelimitedDuration(item.timeInRecording / 1000, fixedUnits)}
```

Comment:
I do think the LOAD text was probably confusing and repetitive, but I'm also not sure that this is the _best_ solution. It's weird to see all these negative timestamps:

![image](https://github.com/PostHog/posthog/assets/18598166/39eb93d4-3d5b-456f-ad44-502aae06ca08)

I feel like it would be better to have a section that is "pre-load events" and a section that is "recording events," or something of the sort. When the recording loads it scrolls automatically to the "recording events" section, but if someone wants to see what happened before recording they can view the "pre-load events."

I do realize that this is out of scope for this PR but just figured I'd mention it so you can keep it in your back pocket.